README.md

```diff
@@ -1,14 +1,32 @@
 # setup-go
 
 <p align="left">
-  <a href="https://github.com/actions/setup-go"><img alt="GitHub Actions status" src="https://github.com/actions/setup-go/workflows/Main%20workflow/badge.svg"></a>
+  <a href="https://github.com/actions/setup-go/actions"><img alt="GitHub Actions status" src="https://github.com/actions/setup-go/workflows/build-test/badge.svg"></a>
+
+  <a href="https://github.com/actions/setup-go/actions"><img alt="versions status" src="https://github.com/actions/setup-go/workflows/go-versions/badge.svg"></a>  
 </p>
 
 This action sets up a go environment for use in actions by:
 
 - optionally downloading and caching a version of Go by version and adding to PATH
 - registering problem matchers for error output
 
+# V2 Beta
+
+The V2 beta offers:
+- Proxy Support
+- stable input 
+- Bug Fixes (including issues around version matching and semver)
+
+```yaml
+steps:
+- uses: actions/checkout@v2
+- uses: actions/setup-go@v2-beta
+  with:
+    go-version: '^1.13.1' # The Go version to download (if necessary) and use.
+- run: go version
+```
+
 # Usage
 
 See [action.yml](action.yml)
```

Comment:
nit: Consider adding the inputs to the readme, rather than pointing to action yaml. I have a script that can keep the two up to date (see [gendocs](https://github.com/actions/checkout/blob/master/package.json#L12) command in package.json; workflow validates `git diff` is empty)